mjpc/estimators/batch.cc

```diff
@@ -106,13 +109,19 @@ void Batch::Initialize(const mjModel* model) {
 
   // dimension
   int nq = model->nq, nv = model->nv, na = model->na;
-  int ntotal = nv * max_history_;
+  int nconfig = nv * max_history_;
   int nsensortotal = nsensordata_ * max_history_;
+  int ntotal = nconfig + nparam_;
 
   // state dimensions
   nstate_ = nq + nv + na;
   ndstate_ = 2 * nv + na;
 
+  // problem dimensions
+  nconfig_ = nv * configuration_length_;
```

Comment:
you just defined this variable 10 lines above, and now you're changing it?  This is confusing...

Reply:
Renamed these to _max. We use these temporary variables (`ntotal_max`, `nconfig_max`) to preallocate memory.